tests/network/sriov.go

```diff
@@ -697,6 +715,34 @@ func waitVMI(vmi *v1.VirtualMachineInstance) (*v1.VirtualMachineInstance, error)
 	return virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &k8smetav1.GetOptions{})
 }
 
+// deleteVMI deletes the specified VMI and waits for its absence.
+// Waiting for the VMI removal is placed intentionally for VMI/s with SR-IOV networks in order
+// to assure resources (VF/s) are fully released before reused again on a new VMI.
+func deleteVMI(vmi *v1.VirtualMachineInstance) error {
+	virtClient, err := kubecli.GetKubevirtClient()
+	if err != nil {
+		panic(err)
```

Comment:
maybe Expect instead?

Reply:
Client fetching has always been treated with panic.
This has been also raised in the refactoring PR, and the answer is the same.. Failing to fetch the client is a test failure, not the tested subject error.

Reply:
Note: I see mixed styles fwiw
sometimes panic, sometimes Expect
and even one time it just skip tests (wrong) here `GetKubernetesVersion`

Reply:
I think panicking might be a better action to take, since I think there's little chance the subsequent tests can create a working client afterwards.